lib/td/command/common.rb

```diff
@@ -41,6 +41,13 @@ def get_client(opts={})
       opts[:retry_post_requests] = Config.retry_post_requests
     end
 
+    # SSL verification options
+    if Config.ssl_verify == false
+      opts[:verify] = false
+    elsif Config.ssl_ca_file
+      opts[:verify] = Config.ssl_ca_file
+    end
```

Comment:
Why can we unify `ssl_verify` and `ssl_ca_file` into a single `ssl_verify` option similar to td-client-ruby's `verify` option?

After all, if `ssl_verify` is false, `ssl_ca_file` value does not apply anyway.